lib/dialects/abstract/query-generator.js

```diff
@@ -1484,48 +1483,68 @@ const QueryGenerator = {
   getQueryOrders(options, model, subQuery) {
     const mainQueryOrder = [];
     const subQueryOrder = [];
+    const validOrderOptions = [
+      'ASC',
+      'DESC',
+      'ASC NULLS LAST',
+      'DESC NULLS LAST',
+      'ASC NULLS FIRST',
+      'DESC NULLS FIRST',
+      'NULLS FIRST',
+      'NULLS LAST'
+    ];
+
+    const methodOrder = order => {
```

Comment:
Well, I think I'll just remove this all together in favor of just checking if the `_isSequelizeMethod` property is set.